CONTRIBUTING.md

```
@@ -1,14 +1,39 @@
 As of `2026-06-09` the following stemcell lines / branches are supported:
 - Ubuntu Jammy / `ubuntu-jammy`
 - Ubuntu Noble / `ubuntu-noble`
 - Ubuntu Resolute / `ubuntu-resolute`
+
+## Merge-Forward Strategy
+
+This repository uses a **merge-forward** strategy to keep stemcell branches in sync.
+
+**Branch order (oldest → newest):**
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)